autoload/neomake/makers/ft/sh.vim

```diff
@@ -10,6 +10,15 @@ function! neomake#makers#ft#sh#shellcheck()
         \ 'errorformat':
             \ '%f:%l:%c: %trror: %m,' .
             \ '%f:%l:%c: %tarning: %m,' .
-            \ '%f:%l:%c: %tote: %m'
+            \ '%f:%l:%c: %tote: %m',
+        \ 'postprocess':
+            \ function('neomake#makers#ft#sh#ShellcheckEntryProcess')
         \ }
 endfunction
+
+function! neomake#makers#ft#sh#ShellcheckEntryProcess(entry)
+    if a:entry.type ==? 'N'
+        let a:entry.type = 'W'
+    endif
+    return a:entry
+endfunction
```

Comment:
@paretje 
> `N` is not a valid entry type, because of which it is marked as an error.

Where is this marked as an error?

I think this should become `I` instead?!